.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-  - 1.24.1
+  - 1.26.2
```

Comment:
Bumping up the (maybe) minimal supported version, is it okay?

Reply:
This is about `...` being deprecated, right?

Reply:
Since fb840836d6c0d30a6ddda1db4ca300c8f952d65a and f5ea106bf11eb4c50ef2c0c46c8729648fcadaeb already bumped the minimum version since last release, I guess it doesn't matter greatly?

Reply:
Just should remember to release the next as `0.5`, a good time for breaking changes. Any other cleanup that is blocked on version compatibility?

Reply:
Yea, this is for `..=` syntax. And we may be able to use `copied()` here by bumping up the version more. But it's not a strong option, I think.

https://github.com/image-rs/inflate/blob/370b79ff96a69e15ff08896ba40d306669bb1f41/src/utils.rs#L25

Reply:
Isn't `bytes: &[u8]`? Then this should be `extend_from_slice`.

Reply:
Good point. It works fine at a quick glance.

Reply:
Could we go to merge?